public/mobile-app/src/routes/page.svelte.test.ts

```diff
@@ -1,11 +1,71 @@
-import { describe, test, expect } from 'vitest'
+import { describe, test, expect, vi } from 'vitest'
 import '@testing-library/jest-dom/vitest'
-import { render, screen } from '@testing-library/svelte'
+import { render, screen, waitFor } from '@testing-library/svelte'
 import Page from './+page.svelte'
 
 describe('/+page.svelte', () => {
-  test('should render h1', () => {
+  test('should render h1 without user email when email and pushSubscription are not set', () => {
+    // When
     render(Page)
-    expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument()
+
+    // Then
+    const title = screen.getByRole('heading', { level: 1 })
+    expect(title).toHaveTextContent("Bienvenue sur l'application AMI")
+  })
+
+  test('should render h1 with user email when email and pushSubscription are set', () => {
+    // Given
+    window.localStorage.setItem('emailLocalStorage', 'test@email.fr')
+    window.localStorage.setItem('pushSubscriptionLocalStorage', '{}')
+
+    // When
+    render(Page)
+
+    // Then
+    const title = screen.getByRole('heading', { level: 1 })
+    expect(title).toHaveTextContent("Bienvenue test@email.fr sur l'application AMI")
```

Comment:
This works 👍 
Another way is to only check that the email is contained in the text, so if there's a typo fix or rewording, the test doesn't need to be rewritten.